meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -14,7 +14,6 @@ RDEPENDS:${PN}:append:mender-growfs-data:mender-systemd:mender-partlabel = " uti
 RDEPENDS:${PN}:append:mender-growfs-data:mender-systemd:mender-partuuid = " util-linux-blkid"
 RDEPENDS:mender-modules-gen = "bash"
```

Comment:
Question is: like we discussed. What to do with this?

You suggested moving it to `mender-artifact`. But having a separate recipe for it is also a good option in my opinion. What do you think?

Reply:
If we just move the `client-cpp` recipe to be the current git recipe, I guess we don't have to do anything really.

Reply:
> You suggested moving it to `mender-artifact`. 

My point was not to move it to `mender-artifact` _recipe_, but that ideally the source of these scripts could have been moved to `mender-artifact` repo and then here it will be natural to be installed by mender-artifact (which provides only a `native` package). Also I don't think you can have one yocto recipe pulling from two different git sources (and if it is possible smells like a very bad practice).

> But having a separate recipe for it is also a good option in my opinion. What do you think?

That would be ideal and if so should also be applied to the current golang recipes. But somewhat is a side little project on itself and it is independent of the cpp client